homeassistant/components/neato/camera.py

```diff
@@ -89,7 +98,7 @@ def update(self):
             return
 
         try:
-            image = self.neato.download_map(image_url)
+            image: urllib3.response.HTTPResponse = self.neato.download_map(image_url)
```

Comment:
Import `HTTPResponse` instead of `urllib3`

Reply:
Done